source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Custom Certificates on the Global CDN
+description: For contract customers who need to retain custom TLS certificates, bring your own certificate to the Global CDN to replace legacy HTTPS service.
+earlyaccess: true
+earlynote: This documentation covers features and options not available across the entire platform.
+---
+
+## Access
+
+A white glove concierge service is now available and limited to contract customers, including **Enterprise**, **EDU+**, **Pantheon One**, **Elite**, and **Reseller**, who are using Legacy SSL.
+
+## Upgrade to the Global CDN
```

Comment:
I suggest this section could be polished a bit to make it more concise and focus on the benefits of the new shiny Global CDN and the difference it'll make for their sites.  Users care more about what they'll get out of this, less so that they're currently using technology we've deprecated.